cache/modules/wc_org_cloudfront/distribution.tf

```diff
@@ -0,0 +1,188 @@
+resource "aws_cloudfront_distribution" "wc_org" {
+  enabled          = true
+  retain_on_delete = false
+  is_ipv6_enabled  = true
+  aliases          = var.aliases
+
+  // The primary load balancer for wc.org apps
+  origin {
+    domain_name = var.load_balancer_dns
+    origin_id   = local.alb_origin_id
+
+    custom_origin_config {
+      origin_protocol_policy = "https-only"
+      http_port              = "80"
+      https_port             = "443"
+      origin_ssl_protocols   = ["TLSv1.2"]
+    }
+  }
+
+  // The S3 bucket containing some static assets
+  origin {
+    domain_name = var.assets_origin.bucket_endpoint
+    origin_id   = local.assets_origin_id
+  }
+
+  # Don't cache 404s
```

Comment:
Do we want the same behaviour for 500 errors? Why are 404s special?

Reply:
Not sure really - this was copied across. I'm going to look at what's _actually_ being cached next because there's some weird stuff going on with no-cache headers being sent by the apps that I would like to understand...